articles/151_node_to_participant_mapping.md

```diff
@@ -0,0 +1,213 @@
+---
+layout: default
+title: Node to Participant mapping
+permalink: articles/node_to_participant_mapping.html
+abstract: This article analyze the performance implications of having a one-to-one Node to Participant mapping and propose an alternative approach.
+  How Nodes are mapped to DDS participants.
+author: '[Ivan Paunovic](https://github.com/ivanpauno)'
+published: true
+categories: Middleware
+---
+
+{:toc}
+
+# {{ page.title }}
+
+<div class="abstract" markdown="1">
+{{ page.abstract }}
+</div>
+
+Original Author: {{ page.author }}
+
+## Introduction
+
+### What is a `Node`?
+
+In ROS, a `Node` is an entity used to group other entities.
```

Comment:
@ivanpauno hmm I wouldn't say it groups other _entities_, as opposed to _interfaces_.